gradle.properties

```diff
@@ -1,2 +1,2 @@
 #Fri, 19 Aug 2016 12:03:02 +0200
-projectVersion=7.16.0-SNAPSHOT
+projectVersion=7.18.0-SNAPSHOT
```

Comment:
yes it should not. that is an outdated diff...i don t know why you can see it :)